tools/ci_build/build.py

```diff
@@ -2009,6 +2009,11 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                 )
 
                 if not args.skip_onnx_tests:
+                    # Disable python onnx tests for CANN EP, because many tests are
```

Comment:
I think you could move check for use_cann to line 2004 (along with tensorrt?)
if it's skipped here, it's calling a bunch of stuff that's not needed e.g. gen_test_models.py in line 2004

Reply:
Got it, I will submit a new commit to fix it. Thanks you a lot.

Reply:
Hi, the new commit is ready, and the PR about the [document ](https://github.com/microsoft/onnxruntime/pull/13875) is also ready, Big thanks for your help.